packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1521,4 +1527,67 @@ void main() {
     expect((mergeableMaterial.children.first as MaterialSlice).color, firstPanelColor);
     expect((mergeableMaterial.children.last as MaterialSlice).color, secondPanelColor);
   });
+
+  testWidgets('ExpansionPanelList icon color test', (WidgetTester tester) async {
+    const Color expandedIconColor = Colors.orange;
+    const Color iconColor = Colors.red;
+
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: SingleChildScrollView(
+          child: SimpleExpansionPanelListTestWidget(
+            iconColor: iconColor,
+            expandedIconColor: expandedIconColor,
+          ),
+        ),
+      ),
+    );
+
+    IconButton iconButton = tester.widget(find.byType(IconButton).at(0));
+    expect(iconButton.color, iconColor);
+
+    await tester.tap(find.byType(ExpandIcon).at(0));
+    await tester.pumpAndSettle();
+
+    iconButton = tester.widget(find.byType(IconButton).at(0));
+    expect(iconButton.color, expandedIconColor);
+  });
+
+  testWidgets('ExpansionPanelRadio icon color test', (WidgetTester tester) async {
+    const Color expandedIconColor = Colors.orange;
+    const Color iconColor = Colors.red;
+
+    final List<ExpansionPanel> _demoItemsRadio = <ExpansionPanelRadio>[
+      ExpansionPanelRadio(
+        headerBuilder: (BuildContext context, bool isExpanded) {
+          return const Text('A');
+        },
+        body: const SizedBox(height: 100.0),
+        value: 0,
+      ),
+    ];
+
+    final ExpansionPanelList _expansionListRadio = ExpansionPanelList.radio(
+      children: _demoItemsRadio,
+      iconColor: iconColor,
+      expandedIconColor: expandedIconColor,
+    );
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: SingleChildScrollView(
+          child: _expansionListRadio,
+        ),
+      ),
+    );
+
+    IconButton iconButton = tester.widget(find.byType(IconButton).at(0));
```

Comment:
Since there will be only one `IconButton` in the tree, you don't need to mention `at`. You can remove it from here and below 2 places too.

```suggestion
    IconButton iconButton = tester.widget(find.byType(IconButton));
```